File: src/main/java/io/gittul/domain/comment/dto/CommentCreateRequest.java
package io.gittul.domain.comment.dto;

import io.gittul.global.validation.OptionalURL;
import jakarta.validation.constraints.Size;

public record CommentCreateRequest(
        @Size(min = 1, max = 200)
        String content,

        @OptionalURL
        String image
) {
}

File: app/src/main/kotlin/io/gittul/app/domain/comment/dto/CommentCreateRequest.kt
package io.gittul.app.domain.comment.dto

import io.gittul.core.global.validation.OptionalURL
import jakarta.validation.constraints.Size

data class CommentCreateRequest(
    val postId: Long,

    @field:Size(min = 1, max = 200)
    val content: String,

    @field:OptionalURL
    val image: String?
)
